docs/dev/development-process/development-process.rst

```diff
@@ -0,0 +1,203 @@
+*******************
+Development Process
+*******************
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+
+The Artemis Development Process Model
+===========================================
+
+.. figure:: ./artemis-process-model.png
+    :align: center
+    :alt: Artemis Process Model - Activity Diagram
+    :figclass: align-center
+
+1. Submit a Feature Request
+---------------------------
+The initial step in our development process involves the creation of a feature request, which is accomplished through the submission of a GitHub Issue. 
+This action can be performed by any stakeholder, including developers, users, or maintainers. 
+The feature request should include a detailed description of the desired functionality, as well as any relevant information that may be useful to the development team. 
+This information may include the rationale for the feature, the expected benefits, and any potential risks or challenges that may be associated with the implementation of the feature.
```

Comment:
> may include

Using "may" in formal documents is considered poor style because its meaning is ambiguous. I recommend using a stronger word here, such as "should".